<ui version="4.0" >
 <class>PaymentDlg</class>
 <widget class="QDialog" name="PaymentDlg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>399</width>
    <height>276</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Payment Form</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="3" column="0" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>381</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" >
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab" >
      <attribute name="title" >
       <string>Cas&amp;h</string>
      </attribute>
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QCheckBox" name="paidCheckBox" >
         <property name="text" >
          <string>&amp;Paid</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2" >
      <attribute name="title" >
       <string>Chec&amp;k</string>
      </attribute>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="1" column="3" >
        <widget class="QLineEdit" name="sortCodeLineEdit" />
       </item>
       <item row="1" column="2" >
        <widget class="QLabel" name="label_8" >
         <property name="text" >
          <string>S&amp;ort Code:</string>
         </property>
         <property name="buddy" >
          <cstring>sortCodeLineEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="3" >
        <widget class="QLineEdit" name="bankLineEdit" />
       </item>
       <item row="0" column="2" >
        <widget class="QLabel" name="label_7" >
         <property name="text" >
          <string>&amp;Bank:</string>
         </property>
         <property name="buddy" >
          <cstring>bankLineEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QLineEdit" name="accountNumLineEdit" />
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="label_6" >
         <property name="text" >
          <string>A&amp;ccount No.:</string>
         </property>
         <property name="buddy" >
          <cstring>accountNumLineEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QLineEdit" name="checkNumLineEdit" />
       </item>
       <item row="0" column="0" >
        <widget class="QLabel" name="label_2" >
         <property name="text" >
          <string>Check &amp;No.:</string>
         </property>
         <property name="buddy" >
          <cstring>checkNumLineEdit</cstring>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3" >
      <attribute name="title" >
       <string>Credit Car&amp;d</string>
      </attribute>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="0" column="1" colspan="3" >
        <widget class="QLineEdit" name="creditCardLineEdit" />
       </item>
       <item row="0" column="0" >
        <widget class="QLabel" name="label_11" >
         <property name="text" >
          <string>&amp;Number:</string>
         </property>
         <property name="buddy" >
          <cstring>creditCardLineEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="3" >
        <widget class="QDateEdit" name="expiryDateEdit" >
         <property name="alignment" >
          <set>Qt::AlignRight</set>
         </property>
         <property name="displayFormat" >
          <string>MMM yyyy</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2" >
        <widget class="QLabel" name="label_10" >
         <property name="text" >
          <string>E&amp;xpiry Date</string>
         </property>
         <property name="buddy" >
          <cstring>expiryDateEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QDateEdit" name="validFromDateEdit" >
         <property name="alignment" >
          <set>Qt::AlignRight</set>
         </property>
         <property name="displayFormat" >
          <string>MMM yyyy</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="label_9" >
         <property name="text" >
          <string>&amp;Valid From:</string>
         </property>
         <property name="buddy" >
          <cstring>validFromDateEdit</cstring>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="0" >
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="0" column="2" >
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>&amp;Surname:</string>
       </property>
       <property name="buddy" >
        <cstring>surnameLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="3" >
      <widget class="QDoubleSpinBox" name="amountSpinBox" >
       <property name="alignment" >
        <set>Qt::AlignRight</set>
       </property>
       <property name="prefix" >
        <string>$ </string>
       </property>
       <property name="maximum" >
        <double>999999.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="0" >
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>&amp;Forename:</string>
       </property>
       <property name="buddy" >
        <cstring>forenameLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="label_5" >
       <property name="text" >
        <string>&amp;Invoice No.:</string>
       </property>
       <property name="buddy" >
        <cstring>invoiceNumSpinBox</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="3" >
      <widget class="QLineEdit" name="surnameLineEdit" />
     </item>
     <item row="0" column="1" >
      <widget class="QLineEdit" name="forenameLineEdit" />
     </item>
     <item row="1" column="2" >
      <widget class="QLabel" name="label_4" >
       <property name="text" >
        <string>&amp;Amount:</string>
       </property>
       <property name="buddy" >
        <cstring>amountSpinBox</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QSpinBox" name="invoiceNumSpinBox" >
       <property name="alignment" >
        <set>Qt::AlignRight</set>
       </property>
       <property name="maximum" >
        <number>999999999</number>
       </property>
       <property name="minimum" >
        <number>1000000</number>
       </property>
       <property name="value" >
        <number>1000000</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>forenameLineEdit</tabstop>
  <tabstop>surnameLineEdit</tabstop>
  <tabstop>invoiceNumSpinBox</tabstop>
  <tabstop>amountSpinBox</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>paidCheckBox</tabstop>
  <tabstop>checkNumLineEdit</tabstop>
  <tabstop>accountNumLineEdit</tabstop>
  <tabstop>bankLineEdit</tabstop>
  <tabstop>sortCodeLineEdit</tabstop>
  <tabstop>creditCardLineEdit</tabstop>
  <tabstop>validFromDateEdit</tabstop>
  <tabstop>expiryDateEdit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PaymentDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>259</x>
     <y>258</y>
    </hint>
    <hint type="destinationlabel" >
     <x>145</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PaymentDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>362</x>
     <y>250</y>
    </hint>
    <hint type="destinationlabel" >
     <x>382</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
